xml/art_sle_ha_install_quick.xml

```diff
@@ -580,37 +580,66 @@ Sep 22 17:01:00 &node1; sbd: [13412]: info: Received command test from &node1;</
+   You now have a running one-node cluster. To view it, proceed as follows:
   </para>
 
-  <important>
-   <title>Secure Password</title>
-   <para>
-    The bootstrap procedure creates a Linux user named
-    <systemitem class="username">hacluster</systemitem> with the password
-    <literal>linux</literal>. You need it for logging in to &hawk2;.
-    Replace the default password with a secure one as soon as possible:
-   </para>
-   <screen>&prompt.root;<command>passwd</command> hacluster</screen>
-  </important>
-  <remark>toms 2016-07-22: Fix screenshot to show alice and bob</remark>
+  <procedure xml:id="pro.ha.inst.quick.hawk2.login">
+   <title>Logging In to the &hawk2; Web Interface</title>
+   <step>
+    <para> On any machine, start a Web browser and make sure that JavaScript and
+     cookies are enabled. </para>
+   </step>
+   <step>
+    <para> As URL, enter the IP address or host name of any cluster node running
+     the &hawk; Web service. Alternatively, enter the address of the virtual
+     IP address that you configured in <xref linkend="step.ha-cluster-init.ip"/>
+     of <xref linkend="pro.ha.inst.quick.setup.ha-cluster-init"/>: </para>
+    <screen>https://<replaceable>HAWKSERVER</replaceable>:7630/</screen>
+    <note>
+     <title>Certificate Warning</title>
+     <para> If a certificate warning appears when you try to access the URL for
+      the first time, a self-signed certificate is in use. Self-signed
+      certificates are not considered trustworthy by default. </para>
+     <para> Ask your cluster operator for the certificate details to verify the
+      certificate. </para>
+     <para> To proceed anyway, you can add an exception in the browser to bypass
+      the warning. </para>
+     <!--FIXME: HAWK1 - see https://bugzilla.suse.com/show_bug.cgi?id=979095
+      <para> For information on how to replace the self-signed certificate with a
+      certificate signed by an official Certificate Authority, refer to
+      <xref linkend="vle.ha.hawk.certificate"/>.</para>-->
+    </note>
+   </step>
+   <step>
+    <para> On the &hawk2; login screen, enter the
+      <guimenu>Username</guimenu> and <guimenu>Password</guimenu> of the
+      <systemitem class="username">hacluster</systemitem> user (or of any other
+     user that is a member of the <systemitem class="groupname"
+      >haclient</systemitem> group). </para>
+    <important>
+     <title>Secure Password</title>
+     <para>
+      The bootstrap procedure creates a Linux user named
+      <systemitem class="username">hacluster</systemitem> with the password
+      <literal>linux</literal>. <!--You need it for logging in to &hawk2;.-->
+      Replace the default password with a secure one as soon as possible:
+     </para>
+     <screen>&prompt.root;<command>passwd</command> hacluster</screen>
+    </important>
+   </step>
+   <step>
+    <para> Click <guimenu>Log In</guimenu>.</para>
```

Comment:
Here I would add one more sentence like the following: 
After login, the &hawk2; Web interface shows the <guimenu>Status</guimenu> screen by default, displaying the current cluster status at a glance:
status at a glance: